ynr/apps/sopn_parsing/helpers/parse_tables.py

```diff
@@ -209,6 +212,14 @@ def get_party(description_model, description, sopn):
     except Party.DoesNotExist:
         party_obj = None
 
+    # Last resort attempt - look for the most similar party object to help when
+    # parsed name is missing a whitespace e.g. Barnsley IndependentGroup
+    party_obj = (
+        qs.annotate(similarity=TrigramSimilarity("name", party_name))
+        .order_by("-similarity")
+        .first()
+    )
+
```

Comment:
This is so much cleaner :)

Reply:
Tested ✅ 
`Check the SOPN at https://candidates.democracyclub.org.uk/elections/local.barnsley.hoyland-milton.2021-05-06/sopn/.
{'name': 'John Brian Evans', 'party_id': 'PP63'}
{'name': 'Glenn Charles Lawrence', 'party_id': 'PP90'}
{'name': 'Daniel Simpson', 'party_id': 'ynmp-party:2'}
{'name': 'James Michael Stowe', 'party_id': 'PP53'}
{'name': 'Angela Waller', 'party_id': 'PP804'}
{'name': 'David Andrew White', 'party_id': 'PP52'}`